mungegithub/mungers/lgtm_handler.go

```diff
@@ -83,7 +98,46 @@ func (LGTMAfterCommitMunger) Munge(obj *github.MungeObject) {
 	}
 }
 
-func (LGTMAfterCommitMunger) isStaleComment(obj *github.MungeObject, comment *githubapi.IssueComment) bool {
+func hasValidReviwer(reviewers []*githubapi.User) bool {
+	for _, r := range reviewers {
+		if r != nil && r.Login != nil {
+			return true
+		}
+	}
+	return false
+}
+
+func foundLGTMFromReviewer(comments []*githubapi.IssueComment, reviewers []*githubapi.User, lastModified *time.Time) bool {
+	for _, c := range comments {
+		if lastModified == nil || c.CreatedAt == nil || (*lastModified).After(*c.CreatedAt) {
+			continue
+		}
+		if !isReviewer(c.User, reviewers) {
```

Comment:
It would be useful to explicitly comment that an LGTM from a reviewer is still needed, and notify the assignee if that person is a reviewer.

cc @apelisse